includes/admin/tools/export/give-export-donations-exporter.php

```diff
@@ -117,6 +117,8 @@ private function get_cols( $columns ) {
 			switch ( $key ) {
 				case 'donation_id' :
 					$cols['donation_id'] = __( 'Donation ID', 'give' );
+				case 'seq_id' :
+					$cols['seq_id'] = __( 'Payment Number', 'give' );
```

Comment:
@raftaar1191 This must say `Donation Number`

Reply:
Updated @ravinderk